comp

```diff
@@ -1,4 +1,27 @@
 #!/bin/sh
+{
+	for i in * /bin/* /usr/bin/* /usr/local/bin/* ~/bin/* ~/.local/bin/*
```

Comment:
I once insisted on using one-letter variables names, but it can obfuscate your code so as to make readability much harder. I suggest something like `file` or `cur_file`, here, assuming you're going with all lowercase.

Reply:
I don’t insist on using single letter parameter names, but naming the `for` loop parameter `i` comes quite naturally.

Reply:
That's fair.